Rakefile

```diff
@@ -7,6 +7,15 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.rspec_opts = "--options #{File.expand_path(".rspec_ci", __dir__)}" if ENV['CI']
 end
 
+desc "Run RSpec code examples (with local postgres dependencies)"
+RSpec::Core::RakeTask.new("spec:dev") do |t|
```

Comment:
Why `spec:dev` for the name here? Maybe `spec:full` or something that indicates that we are including additional tests?

Reply:
Because these are specific changes that are for a developers machine, where `spec:ci` is the CI specific changes.

Now, in retrospect, seeing that it wasn't super straight forward to get this working on Travis like I was hoping, I almost think it makes more sense to have local and CI doing the same process with `pg_ctl` instead.  But unsure if that is worth the effort at this point since there are a bunch of other PRs dependent on this one I have in the pipeline.

Reply:
> Why spec:dev for the name here?

Because this was meant to specify that this specifically designed to run on a development machine, where it would be creating a separate instance of postgres as part of the test run to do it.  I feel like the `:default` task is the `spec:full` task as you are describing it.

Reply:
Yeah, that makes sense in context of the ci task. I was reviewing this commit-by-commit so at the time this was created the naming seemed odd.